script.js

```diff
@@ -17,3 +18,246 @@ for (let i = 0; i < mobileMenuIcons.length; i += 1) {
     mobileMenuIcons[1].classList.toggle('hide');
   });
 }
+
+const projects = [
+  {
+    name: 'Tonic',
+    dept: {
+      name: 'canopy',
+      role: 'Back End Dev',
+      year: 2015,
+    },
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'images/work1.png',
+    technologies: [
+      'html',
+      'css',
+      'javaScript',
+    ],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+  },
+
+  {
+    name: 'Multi-Post Stories',
+    dept: {
+      name: 'canopy',
+      role: 'Back End Dev',
+      year: 2015,
+    },
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'images/work2.png',
+    technologies: [
+      'html',
+      'css',
+      'javaScript',
+    ],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+  },
+
+  {
+    name: 'Tonic',
+    dept: {
+      name: 'canopy',
+      role: 'Back End Dev',
+      year: 2015,
+    },
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'images/work3.png',
+    technologies: [
+      'html',
+      'css',
+      'javaScript',
+    ],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+  },
+
+  {
+    name: 'Multi-Post Stories',
+    dept: {
+      name: 'canopy',
+      role: 'Back End Dev',
+      year: 2015,
+    },
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'images/work4.png',
+    technologies: [
+      'html',
+      'css',
+      'javaScript',
+    ],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+  },
+];
+
+for (let i = 0; i < projects.length; i += 1) {
```

Comment:
- [OPTIONAL] This `for loop` is too long. You should always keep your blocks to at most 20 lines of code. You could break this down by extracting some of the code in the loop into functions. Modular code is easy to debug and maintain. 💡